### ISSUE TITLE
WEB интерфейс

### ISSUE THREAD

**Issue by lti1** [Laxilef/OTGateway#99]
Добрый день!
Пользуюсь управлением через WEB интерфейс, мне этого достаточно. Только столкнулся со сложностью, на компьютере через Chrome страница Дашборд отображается нормально, на смартфоне ни в Chrome, ни в Safri не отображается установленная температура, тумблер активации в положении выключено и нет реакции на кнопки изменения температуры.
Чистил кэш, включал в настройках браузеров "Версия для ПК", менял ориентацию экрана, перегружал ESP отключением питания - ничего не помогает. Хотя ранее все отображалось нормально.
В чем может быть дело?
Спасибо!
![image](https://github.com/user-attachments/assets/be08747e-b767-4781-8eae-c6e6bb205601)
![image](https://github.com/user-attachments/assets/f211e1da-72c1-4edd-8bd1-33e0ec1fb73e)

**Comment by Laxilef** [comment]
Здравствуйте.
1. Какая версия прошивки?
2. Какое разрешение экрана у устройства?

> Хотя ранее все отображалось нормально.

Попробуйте открыть в приватной вкладке, поменялось ли что-то?

**Comment by lti1** [comment]
1. 1.4.6
2. iPhone 13, 2532×1170
`Да, не понятно.`

**Comment by lti1** [comment]
> 2. Какое разрешение экрана у устройства?
За смартфоне установлен увеличенный интерфейс, сейчас попробовал установить стандартный, отображение текущей температуры выровнялось по центру, но так же не отображается установленная температура, тумблер активации.
![image](https://github.com/user-attachments/assets/07f2c036-3c12-4fb3-8996-4ca361dbc26d)

**Comment by Laxilef** [comment]
Айфонами не пользуюсь, попробую найти для тестов. Проблема скорее всего в верстке.
В альбомной ориентации экрана так же себя ведет интерфейс?

**Comment by lti1** [comment]
> В альбомной ориентации экрана так же себя ведет интерфейс?

Да, так же.
Спасибо!

**Comment by Laxilef** [comment]
> Да, так же.

А вот это странно :( Предполагал, что на экран не влезает просто.

**Comment by lti1** [comment]
Все остальные страницы подгружаются нормально, иногда бывает не раскрываются подпункты настроек, это скорее всего какая-то иная причина, например инет подтормаживает или роутер что-то не пропускает.
По моему самое странное, что тумблер включения отопления на компьютере во включённом состоянии, а на телефоне в выключенном, в одно и тоже время и нет никакой реакции на кнопки изменения температуры на телефоне. То есть если я открою страницу сразу и там, и там, на телефоне буду менять температуру, хоть её и не видно, или изменять положение тумблера, то на компьютере никаких изменений температуры и тумблера не происходит, пробовал ждать более минуты.

**Comment by lti1** [comment]
Поэкспериментировал, открыл страницу управления во встроенном браузере telegram, в нем отображается и управляется все нормально.
Похоже, что дело в Chrome и Safari.
Правда поймал после этого, что не отображалась температура на компьютере, но после обновления страницы все стало нормально.
![image](https://github.com/user-attachments/assets/8873d54e-d96e-45a5-9cae-f54f69dfb7a6)
![image](https://github.com/user-attachments/assets/c7ca93ab-7cb6-468b-ad95-a2d36a91cda9)

**Comment by Laxilef** [comment]
А есть возможность прошить бету?
Esp какая?

**Comment by lti1** [comment]
> А есть возможность прошить бету?

Да, смогу в выходные доехать. По воздуху у меня почему-то не проходит обновление, ошибку выдает.

> Esp какая?

S2 mini
![image](https://github.com/user-attachments/assets/6b077a46-7b3a-4381-a72a-9c2a5a462448)

**Comment by Laxilef** [comment]
> По воздуху у меня почему-то не проходит обновление, ошибку выдает.

Какую?

**Comment by lti1** [comment]
> Какую?

![image](https://github.com/user-attachments/assets/3341701b-8ba4-4ca0-926b-73a0d0b115d2)

**Comment by Laxilef** [comment]
https://t.me/otgateway/6682

**Comment by lti1** [comment]
> А с ПК?

Точно не помню, пробовал или нет. Попробую по воздуху ещё раз с компа, когда поеду заливать бету.

**Comment by lti1** [comment]
> А есть возможность прошить бету?

К сожалению обновление не помогло.
![image](https://github.com/user-attachments/assets/d2071720-3682-4423-b471-237e0c96314d)

Повесил для эксперимента на роутер ещё ESP32_C3 не подключенную к котлу, с прошивкой 1.4.5, на ней пока все отображается нормально.

**Comment by Laxilef** [comment]
Проверил 1.5.0 через browserstack на iphone 13, проблема не воспроизводится
![Screenshot_1](https://github.com/user-attachments/assets/6b0d56bb-05fa-4f8c-b7fb-c6581d364712)

**Comment by lti1** [comment]
Могу предоставить доступ к управлению моего котла.
Нашел у себя старенький планшет, на Андройд отображается нормально.

**Comment by lti1** [comment]
[Laxilef](https://github.com/Laxilef), я нашел какая настройка влияет на отображение установленной температуры и работу тумблера активации - это Настройки портала -> Требовать аутентификацию. Если отключить аутентификацию, то на iPhone все работает как на ПК и Андройд! К сожалению у себя эту настройку отключить не могу, так как подключаюсь удаленно.
![image](https://github.com/user-attachments/assets/00bf8575-94d1-45fd-ab04-1c48b044d1ce)

**Comment by Laxilef** [comment]
Интересно, поэкспериментирую с аутентификацией, спасибо.

**Comment by lti1** [comment]
[Laxilef](https://github.com/Laxilef), спасибо! Похоже помогло.

**Comment by Laxilef** [comment]
С ios пытались прошить?

**Comment by lti1** [comment]
> С ios пытались прошить?

Нет, с ПК на Windows 11, через Chrome.

> рекомендую проверить.

[Laxilef](https://github.com/Laxilef), спасибо! Да, слетели настройки датчиков, настроил сразу после обновления и восстановления настроек.

**Comment by Laxilef** [comment]
Попробуйте по ОТА обновиться с альфы на [релиз](https://github.com/Laxilef/OTGateway/releases/tag/1.5.0). Сохраняется ошибка?

**Comment by lti1** [comment]
> Попробуйте по ОТА обновиться с альфы на [релиз](https://github.com/Laxilef/OTGateway/releases/tag/1.5.0). Сохраняется ошибка?

Попробовал, ошибки нет, пишет что обновление прошло нормально, но в системе показывает Версия 1.5.0-alpha, Дата: Dec 2 2024 06:32:03.

![image](https://github.com/user-attachments/assets/8437ee54-647e-49e8-b72b-f289c0c8c012)

**Comment by lti1** [comment]
> И после перезагрузки есп не обновилось?

Нет, тоже самое.

**Comment by lti1** [comment]
Похоже мой "косяк", вычитал сегодня в телеграм, что нужно было шить 1.5.0.bin, а я все это время пытался залить 1.5.0.factory.bin. Теперь корректно версия прошивки отображается.

![image](https://github.com/user-attachments/assets/a7bfeea4-42ea-4cc0-876f-8c0ed6b841a5)
